docs/index.asciidoc

```diff
@@ -20,6 +20,7 @@ include::{include_path}/plugin_header.asciidoc[]
 
 ==== Description
 
+Adds arbitrary fields to an event
```

Comment:
@jsvd Fixed. TBH, I didn't have a lot of time to fix all of the issues in the plugin docs. In the interest of making sure the build-breaking issues got resolved, I used the short descriptions from the overview pages. If you have better descriptions to suggest, that would be great, of course.